Source/ARTStatus.m

```diff
@@ -35,6 +35,9 @@ + (ARTErrorInfo *)createWithCode:(NSInteger)code status:(NSInteger)status messag
 }
 
 + (ARTErrorInfo *)createFromNSError:(NSError *)error {
+    if (!error) {
+        return nil;
+    }
```

Comment:
Now I see why you passed `[ARTStatus createFromNSError: ... ]` in the successful callback above.
I think it's confusing to see a "I successfully got results and here they are" callback that takes a (at first sight) non-null error value. Did you do it so that the syntax looks nicer in Swift?

Reply:
I change it with 861f32b. Thanks  👍